### ISSUE TITLE
Documentation mistake in tf.keras.layers.MaxPool2D python section

### ISSUE THREAD

**Issue by chinya07** [tensorflow/tensorflow#46998]
## URL(s) with the issue: 
https://www.tensorflow.org/api_docs/python/tf/keras/layers/MaxPool2D

## Description of issue (what needs changing):

A Documentation mistake in tf.keras.layers.MaxPool2D python section(example)

### Clear description

There is a documentation mistake in tf.keras.layers.MaxPool2D python section(example). 
In the description part of there are few examples given with their respective code snippets.
Under this text-"For example, for stride=(2,2) and padding="valid":" in the code snippet of example instead of **strides=(2,2)**, **strides=(1,1)** is written.

<img width="939" alt="Screenshot 2021-02-08 at 10 16 05 AM" src="https://user-images.githubusercontent.com/38719024/107178001-33b1de00-69f9-11eb-8c7a-6114457686c0.png">

**Comment by amahendrakar** [comment]
@chinya07,
Thank you for reporting the error. 

The change is being tracked in PR [#47003](https://github.com/tensorflow/tensorflow/pull/47003) and will reflect as soon as it gets merged.
